include/eigen_helper.h

```diff
@@ -9,7 +9,7 @@
 #define EIGEN_DONT_VECTORIZE
 #endif
 
-#include <cmath>
+#include <math.h>
```

Comment:
why this change?

Reply:
Actually, it needs the functions (isinf, isnan) in the root namespace, not std::.

Reply:
What needs the functions in the root namespace?

Reply:
I think this is the cause: https://gitlab.com/libeigen/eigen/-/blob/master/Eigen/src/Core/MathFunctions.h#L984